docs/Checkpoints.md

```diff
@@ -0,0 +1,49 @@
+Checkpoints
+===
+
+There are 3 types of OLMo checkpoints.
+
+1. OLMo (standard) checkpoints. These checkpoints can be produced and used by the code in this repo. "OLMo checkpoints" will typically refer to these checkpoints.
+2. Transformers checkpoints. These checkpoints can be produced and used via the OLMo implementation in the [Hugging Face Transformers](https://huggingface.co/docs/transformers/index) library. As we continue to develop and improve OLMo, our implementation in this repo may temporariy become incompatible with the implementation in the Transformer library.
+3. HF OLMo checkpoints. These checkpoints can be produced and used via the `hf_olmo` package. The `hf_olmo` package provides basic Transformers functionality while always staying compatible with the OLMo library.
+
+OLMo (standard) checkpoints
+---
+
+There are 2 categories of OLMo checkpoints:
+- unsharded: a complete checkpoint in a standard form;
+- sharded: a checkpoint that has been broken down into smaller components, for easier use in our multi-node training.
+
+Unless otherwise specified, an OLMo checkpoint is assumed to be unsharded. OLMo sharded and unsharded checkpoints can run the pretraining and fine-tuning provided in this repo.
+
+#### Sharded OLMo Checkpoints
+
+There are currently 4 types of sharded checkpoints:
+- torch_legacy,
+- torch_new,
+- local,
+- olmo_core.
+
+We are still working on improving sharded checkpointing and thus do not have any guidelines for using them at present. A sharded checkpoint can be converted to an unsharded checkpoint using [unshard.py](https://github.com/allenai/OLMo/blob/main/scripts/unshard.py).
+
+Transformers Checkpoints
+---
+
+These checkpoints can be used with the OLMo implementation in the Transformers library. Since the OLMo implementation is integrated into the library, OLMo models support most Transformers model functionality. These checkpoints cannot run the pretraining or fine-tuning provided in this repo.
```

Comment:
```suggestion
These checkpoints can be used with the OLMo implementation in the Transformers library. Since the OLMo implementation is integrated into the library, OLMo models support most Transformers model functionality. These checkpoints cannot be used with the pretraining/fine-tuning script provided in this repo.
```